### DISCUSSION TITLE
CAPTCHAs for Idle-Game will be removed on 23 June ???

### DISCUSSION THREAD

**Discussion by isidororeyes** [coccoinomane/crabada.py#124] [General]
Hello @coccoinomane,

I think the team is completely lost in terms of strategy...one day they announce anti-bot enforcement and the other remove captchas that are supposed to prevent bots from looting...

I'm lost here. Any ideas guys about the reason behind this? 

On the other side, can you push the create loot functionality back to the main? I'm curious how this will play out.

Best

**Comment by coccoinomane** [comment]
Ciao @isidororeyes and thank you for the update!

I am not sure why the team decided to drop the captcha. Maybe they just wanted to stop paying for the Captcha service?

Let's see how they implement the change at the smart contract level. If they just revert to how it worked before, it would be feasible to re-activate the attack functionality of crabada.py.

Cheers,
Cocco
